### DISCUSSION TITLE
Принять логи из стороннего Linux устройства

### DISCUSSION THREAD

**Discussion by tvideo** [totumonline/totum-ru-issues-and-discussions#122] [Как это сделать?] [answered]
Здравствуйте. 
Подскажите, плз, каким образом можно решить следующую задачку:
Есть устройство Linux которое может транслировать логи на любой IP.
Каким образом можно принять эту трансляцию логов?
или подскажите в каком направлении думать.
Спасибо

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Смотреть в направлении [Remotes](https://ru.docs.totum.online/api#part-4)

Те вы создаете в Тотум точку подключения и отправляете на нее эти логи. В Тотум пишете Тотум-кодом, что с ними делать.
